.github/ISSUE_TEMPLATE/algorand-engineering-team-issue-template.md

```diff
@@ -1,18 +1,9 @@
 ---
-name: Algorand Engineering Team Issue Template
-about: This is the template that Algorand internal team members use in conjunction
-  with their task management process. Feel free to use if you're an external contributor.
-title: ''
-labels: ''
-assignees: ''
+**Requirements Template**
```

Comment:
I think we need to keep the formatting here. (name/about/title/labels/assignees). These are used by Github I think.

Reply:
I'd also leave the name "Algorand Engineering Team Issue Template"